package.json

```diff
@@ -12,7 +12,9 @@
     "lint": "eslint **/*.js",
     "validate": "npm run lint && npm test",
     "test": "jasmine ./test/test.js",
-    "prebuild": "prebuild --all --strip --verbose",
+    "prebuild": "npm run prebuild-node && npm run prebuild-electron",
+    "prebuild-node": "prebuild --force --strip --verbose -t 12.0.0 -t 14.0.0 -t 16.0.0",
+    "prebuild-electron": "prebuild --force --strip --verbose -r electron -t 13.0.0 -t 14.0.0 -t 15.0.0 -t 16.0.0",
```

Comment:
Following up from https://github.com/MadLittleMods/node-usb-detection/pull/165#discussion_r818223445 where we skipped Node.js 17.0.0 builds (fix from https://github.com/serialport/node-serialport/pull/2356), this results in not many prebuilds. 26 vs >100 becaues it's only building for `12.0.0` `14.0.0` and `16.0.0` directly and not the in between versions.

Reply:
The solution is more probably to move to `prebuildify` which is tracked by https://github.com/MadLittleMods/node-usb-detection/issues/154 or making the module context aware/N-API stuff, see https://github.com/MadLittleMods/node-usb-detection/issues/102 and https://github.com/MadLittleMods/node-usb-detection/issues/42